dockstore-webservice/src/main/java/io/dockstore/webservice/core/Workflow.java

```diff
@@ -113,6 +113,10 @@
     @Convert(converter = SourceControlConverter.class)
     private SourceControl sourceControl;
 
+    @Column
```

Comment:
We don't always do this, but I think we should -- can you add `@size(max = 256)` so that this field is in sync with the database column max in the migrations file? But first see my comment about 256 there.